triples-generator/src/test/scala/io/renku/triplesgenerator/events/consumers/tsmigrationrequest/migrations/reprovisioning/ReProvisionJudgeSpec.scala

```diff
@@ -42,59 +41,32 @@ class ReProvisionJudgeSpec extends AnyWordSpec with should.Matchers with MockFac
     "return true when TS schema version cannot be found" in new TestCase {
       (versionPairFinder.find _).expects().returning(Option.empty[RenkuVersionPair].pure[Try])
 
-      judge(NonEmptyList.one(renkuVersionPairs.generateOne)).reProvisioningNeeded() shouldBe true.pure[Try]
+      judge(compatibilityGen.generateOne).reProvisioningNeeded() shouldBe true.pure[Try]
     }
 
     "return true when TS schema version is different than the matrix schema version" in new TestCase {
```

Comment:
Maybe the 'config' word should be used instead of the 'matrix' in the test name.